docs/debugging.md

```diff
@@ -0,0 +1,40 @@
+## Debugging Tutorial
+
+### For debugging purposes the following must be installed :
+
+[Visual Studio Code](https://code.visualstudio.com/)
+[Node.js](https://www.npmjs.com/) - to setup node.js you may follow the instructions in [Environment Setup](https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md)
+
+### Setting up VS Code and Launching Telescope :
+
+To begin debugging one must make sure that the telescope launch.json file can be found as VS code will automatically generate it's own launch.json file if none is found.
+This launch.json file contains **important** debugging configuration information which should be located in the .vscode folder for telescope to run.
+
+Once the launch.json file has been located, you can move on to the "Run View" tab.  
+[VS Run View Screenshot](https://drive.google.com/file/d/1dvXhen8_eT68qEMa8Z6OpejDiMOG1pLN)
```

Comment:
These images are still pointing to Google Drive, please include in the repo and use relative links.